jans-auth-server/server/src/main/java/io/jans/as/server/register/ws/rs/RegisterService.java

```diff
@@ -149,19 +149,19 @@ public void updateClientFromRequestObject(Client client, RegisterRequest request
         }
 
         for (String key : requestObject.getClientNameLanguageTags()) {
-            client.setClientName(requestObject.getClientName(key), Locale.forLanguageTag(key));
+            client.setClientNameLocalized(requestObject.getClientName(key), Locale.forLanguageTag(key));
```

Comment:
So if there is no language tag set we end up client name skipped even if it's set in request object.
I mean we need plain setters for non-localized versions, like 
```
client.setClientName(requestObject.getClientName());
```
Isn't it? Otherwise how non-localized version `io.jans.as.common.model.registration.Client#clientName` is used?